myuw/templates/handlebars/card/student_info.html

```diff
@@ -1,168 +1,208 @@
-                                {{/each}}
-                            {{/if}}
-                        </ul>
+                                    {{/each}}
+                                {{/if}}
+                            </ul>
+                        </div>
                     </div>
                 </div>
-              </div>
             </div>
 
-            <div property="workLocation" typeof="Place"  class="row" style="margin-top: 10px;">
-              <span class="sr-only" property="name">Student Information</span>
-              <div class="profile-local-address col-xs-6">
-                <h4>Local Address</h4>
-                {{#if local_address}}
-                {{#with local_address}}		
-                {{#if street_line2}}		
-                <div><span class="">{{street_line2}}</span></div>		
-                {{/if}}		
-                {{#if street_line1}}		
-                <div><span class="">{{street_line1}}</span></div>		
-                {{/if}}		
-                {{#if city}}<span class="">{{city}}</span>{{/if}}{{#if state}}<span class="">, {{state}}</span>{{/if}}{{#if postal_code}} <span class="">{{postal_code}}</span>{{/if}}{{#if zip_code}} <span class="">{{zip_code}}</span>{{/if}}		
-                {{#if country}}		
-                <div><span class="">{{country}}</span></div>		
-                {{/if}}		
-                {{/with}}		
-                {{#if local_phone}}		
-                <div>Phone: <span class="">{{formatPhoneNumber local_phone}}</span></div>		
-                {{/if}}		
-                {{else}}
-                <span class="my-text-muted">No address available</span>
-                {{/if}}
-              </div>
-
-              <div class="profile-perm-address col-xs-6">
-                <h4>Permanent Address</h4>
-                {{#if permanent_address}}
-                {{#with permanent_address}}
-                {{#if street_line2}}
-                <div><span class="">{{street_line2}}</span></div>
-                {{/if}}
-                {{#if street_line1}}
-                <div><span class="">{{street_line1}}</span></div>
-                {{/if}}
-                {{#if city}}<span class="">{{city}}</span>{{/if}}{{#if state}}<span class="">, {{state}}</span>{{/if}}{{#if postal_code}} <span class="">{{postal_code}}</span>{{/if}}{{#if zip_code}} <span class="">{{zip_code}}</span>{{/if}}
-                {{#if country}}
-                <div><span class="">{{country}}</span></div>
-                {{/if}}
-                {{/with}}
-                {{#if permanent_phone}}
-                <div>Phone: <span class="">{{formatPhoneNumber permanent_phone}}</span></div>
-                {{/if}}
-                {{else}}
-                <span class="my-text-muted">No address available</span>
-                {{/if}}
-              </div>		
+            <div class="myuw-card-section">
+
+                <div property="workLocation" typeof="Place" class="row">
```

Comment:
Remove `property="workLocation"` and 'typeof="Place"' since this is their home address. Can add in proper schema.org meta later.